keyboards/darkproject/kd83a_bfg_edition/config.h

```diff
@@ -0,0 +1,142 @@
+/* Copyright 2022 GSKY <gskyGit@gsky.com.tw>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define DEVICE_VER 0x0001
+#define VENDOR_ID 0x342D
+#define PRODUCT_ID 0xE392
+#define MANUFACTURER GSKY
+#define PRODUCT KD83A_BFG_Edition
+
+/* key matrix size */
+#define MATRIX_ROWS 14
+#define MATRIX_COLS 8
+
+#define MATRIX_ROW_PINS \
+    { B0, B1, B2, B3, B4, B5, B6, B7, B8, B9, B10, B11, B12, B13 }
+#define MATRIX_COL_PINS \
+    { A0, A1, A2, A3, A4, A8, A9, A10 }
+
+#define ENCODERS_PAD_A \
+    { C2 }
+#define ENCODERS_PAD_B \
+    { C3 }
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Hold ESC on start up to clear EEPROM and boot into bootloader mode */
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+#define TAP_CODE_DELAY 10
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* External spi flash */
+#define EXTERNAL_FLASH_SPI_SLAVE_SELECT_PIN B14
+#define RGB_DISABLE_WHEN_USB_SUSPENDED
+#define EXTERNAL_FLASH_SIZE (8 * 1024)
+#define EEPROM_SIZE EXTERNAL_FLASH_SIZE
```

Comment:
```suggestion
#define TRANSIENT_EEPROM_SIZE 8192
```

Reply:
@tzarc Could you please help me out. I have updated the files, Help me to review.